index.js

```diff
@@ -1110,25 +1112,34 @@ class ForwardEmail {
 }
 
 if (!module.parent) {
+  const hostname = process.env.HOSTNAME || 'forwardemail.net';
+  const defaultDeployment = hostname === 'forwardemail.net';
+
   const config = {
-    noReply: 'no-reply@forwardemail.net',
-    exchanges: ['mx1.forwardemail.net', 'mx2.forwardemail.net'],
+    noReply: `no-reply@${hostname}`,
+    exchanges: process.env.EXCHANGES 
+      ? process.env.EXCHANGES.split(',')
+      : (defaultDeployment ?  ['mx1.forwardemail.net', 'mx2.forwardemail.net'] : [hostname]),
     ssl: {},
-    dkim: {}
+    dkim: {},
+    ipAddress: process.env.IP_ADDRESS || undefined,
   };
 
-  if (process.env.NODE_ENV === 'production') {
-    // needsUpgrade = true;
+
+  if (process.env.SECURE === 'true') {
     config.ssl = {
-      secure: process.env.SECURE === 'true',
-      key: fs.readFileSync('/home/deploy/mx1.forwardemail.net.key', 'utf8'),
-      cert: fs.readFileSync('/home/deploy/mx1.forwardemail.net.cert', 'utf8'),
-      ca: fs.readFileSync('/home/deploy/mx1.forwardemail.net.ca', 'utf8')
+      secure: true,
+      key: fs.readFileSync(process.env.SSL_KEY || '/home/deploy/mx1.forwardemail.net.key', 'utf8'),
```

Comment:
Shouldn't this be...
```
key: process.env.SSL_KEY || fs.readFileSync('/home/deploy/mx1.forwardemail.net.key', 'utf8'),
```

Reply:
I guess it depends wether you want to provide it in an env var directly or read it from a file. I think both options are valid. Maybe I should add both. (SSL_KEY and SSK_KEY_FILE)

Reply:
And that way, it will also be easier to add the keys from Docker. You just specify them as env vars when running `docker run`.

Reply:
Fixed